src/OrchardCore/OrchardCore.Module.Targets/OrchardCore.Module.Targets.targets

```diff
@@ -87,16 +87,6 @@
     </ItemGroup>
   </Target>
 
-  <Target Name="UpdateModuleAssets"
```

Comment:
What was it doing?

Reply:
Believe it or not, I don't know :), seems JT did is for a reason when there's `WithCultures()` in the past

Reply:
Then perhaps better not to remove it.

Reply:
I don't think that's related to the WithCulture in the localization API

Reply:
See https://learn.microsoft.com/en-us/visualstudio/msbuild/assignculture-task?view=vs-2022:

> `Files`: Specifies the list of files with embedded culture names to assign a culture to. The task tries to figure out if each file is a culture-specific resource, and if so, what culture. To skip this detection process and force a file to be culture-neutral, set the metadata entry WithCulture to false.

Reply:
This change was introduced in #1483, TBH I never seen such this while working on localization, so I will revert this change and let us move this PR forward